lib/dealership.rb

```diff
@@ -0,0 +1,32 @@
+class Dealership
+  attr_reader :inventory,
+              :inventory_count
+
+  def initialize(name, location)
+    @name = name
+    @location = location
+    @inventory = []
+    @inventory_count = 0
```

Comment:
I would suggest making `inventory_count` a method instead of having it in your `initialize`. It isn't a counter, it is just returning the count of elements in your `inventory` array.
```
def inventory_count
   @inventory.length
end
```

Reply:
thank you very much for this.